sdk/keyvault/keyvault-certificates/package.json

```diff
@@ -83,17 +83,17 @@
   },
   "dependencies": {
     "@azure/abort-controller": "^1.0.0",
-    "@azure/core-http": "^1.0.0",
+    "@azure/core-http": "^1.1.0",
     "@azure/core-lro": "^1.0.0",
     "@azure/core-paging": "^1.0.0",
-    "@azure/core-tracing": "1.0.0-preview.7",
+    "@azure/core-tracing": "1.0.0-preview.8",
     "@azure/logger": "^1.0.0",
-    "@opentelemetry/types": "^0.2.0",
+    "@opentelemetry/api": "^0.6.1",
     "tslib": "^1.10.0"
   },
   "devDependencies": {
     "@azure/eslint-plugin-azure-sdk": "^2.0.1",
-    "@azure/identity": "^1.0.0",
+    "@azure/identity": "1.1.0-preview.3",
     "@azure/keyvault-keys": "^4.0.0",
     "@azure/keyvault-secrets": "^4.0.0",
```

Comment:
certificates has dev dependencies on both keys & secrets, so how is it that only secrets is being updated in this PR?

Reply:
The keys dependency should be removed, it doesn't do anything.

Reply:
I've made a PR to remove this dependency from keyvault-certificates: https://github.com/Azure/azure-sdk-for-js/pull/8874 It's a draft, but it's something inconsequential, so we can go back to that when we have time.

Reply:
Shouldnt the min version of the secrets dependency be updated to 4.0.3 now that the tests wont work with older versions of 4.0.0 - 4.0.2? And if we are making that change to secrets, we might as well remove keys from here